tests/pytest/eligibility/test_views.py

```diff
@@ -71,15 +71,55 @@ def model_EnrollmentFlow_with_form_class(mocker, model_EnrollmentFlow):
     return model_EnrollmentFlow
 
 
+@pytest.fixture
+def model_EnrollmentFlow_only_digital(model_EnrollmentFlow):
+    model_EnrollmentFlow.supported_enrollment_methods = ["digital"]
+    model_EnrollmentFlow.save()
+    return model_EnrollmentFlow
+
+
+@pytest.fixture
+def model_EnrollmentFlow_only_in_person(model_EnrollmentFlow):
+    model_EnrollmentFlow.supported_enrollment_methods = ["in_person"]
+    model_EnrollmentFlow.save()
+    return model_EnrollmentFlow
+
+
+@pytest.mark.django_db
+def test_index_filter_out_in_person_flow(mocked_session_agency, model_EnrollmentFlow_only_in_person, client):
+    path = reverse(routes.ELIGIBILITY_INDEX)
+    response = client.get(path)
+
+    assert mocked_session_agency.return_value.enrollment_flows.count() == 1
+    assert mocked_session_agency.return_value.enrollment_flows.first().supported_enrollment_methods == ["in_person"]
+    assert response.status_code == 200
+    assert "form" in response.context_data
+    assert len(response.context_data["form"].fields["flow"].choices) == 0
+
+
+@pytest.mark.django_db
+def test_index_show_digital_flow(mocked_session_agency, model_EnrollmentFlow_only_digital, client):
+    path = reverse(routes.ELIGIBILITY_INDEX)
+    response = client.get(path)
+
+    assert mocked_session_agency.return_value.enrollment_flows.count() == 1
+    assert mocked_session_agency.return_value.enrollment_flows.first().supported_enrollment_methods == ["digital"]
+    assert response.status_code == 200
+    assert response.status_code == 200
+    assert "form" in response.context_data
+    len(response.context_data["form"].fields["flow"].choices) == 1
```

Comment:
These specs are testing:

- When you have an agency with only 1 flow, and that flow is only supported for `in-person`, the form should have 0 fields. (The flow is filtered out from the digital flow form view).
- When you have an agency with only 1 flow, and that flow is `digital`, the form should have 1 field. (The flow is NOT filtered out from the digital flow form view).

I would have liked to have been able to have an assertion on the last line of these specs that are more like "The response's form should have a field that is supported by digital", but the `context_data["form"].fields["flow"].choices` only returns an id of the flow, not name of the flow or anything more. So I decided to test it like this instead. I'm open to other ideas on how to test this!